doc/changelog.md

```diff
@@ -214,101 +310,37 @@
   This is not so much a breaking change, it was just never used in the past.
   This will be set back to its original behaviour once it is implemented in the frontend.
 
-* We match better the Jax convention for returning gradient, jacobian, vjp and jvp. Therefore some breaking
-  changes about the return shapes of those functions were introduced.
-  [(#500)](https://github.com/PennyLaneAI/catalyst/pull/500)
-  [(#501)](https://github.com/PennyLaneAI/catalyst/pull/501)
-  [(#508)](https://github.com/PennyLaneAI/catalyst/pull/508)
-
-* The entry point name convention has changed.
-  [(#493)](https://github.com/PennyLaneAI/catalyst/pull/493)
-
-  From the [documentation](https://packaging.python.org/en/latest/specifications/entry-points/#data-model):
-
-     Within a distribution, entry points should be unique.
-
-  This meant that within a single distribution there cannot be two different `qjit` entry points like so:
-
-  ```py
-  entry_points={"pennylane.compilers": [
-      "qjit = foo:Foo",
-      "qjit = bar:Bar",
-  ]}
-  ```
-
-  Now, entry point's names are preppended with the name of the compiler.
-
-  ```py
-  entry_points={"pennylane.compilers": [
-      "compilerFoo.qjit = foo:Foo",
-      "compilerBar.qjit = bar:Bar",
-  ]}
-  ```
-
-  The behaviour will be reflected when using the `@qml.qjit` decorator.
-  For example,
-
-  ```py
-  @qml.qjit(compiler="compilerFoo")
-  def circuit(): ...
-  ```
+* Changes to the runtime QIR API and dependencies, to avoid symbol conflicts
```

Comment:
It depends on whether we have consumers of this IR. The intended use case of Catalyst is that device implementors implement the `QuantumDevice` C++ interface and they don't have to worry about QIR. I consider this an internal change at the moment.

But once we start feeding QIR as an input to other devices, this would be considered a breaking change.